app/tools/android/make_apk_test.py

```diff
@@ -906,6 +927,21 @@ def testArch(self):
         self.assertFalse(os.path.isfile('Example_1.0.0._arm.apk'))
       self.assertFalse(os.path.isfile('Example_1.0.0_x86.apk'))
       self.assertFalse(os.path.isfile('Example_1.0.0_x86_64.apk'))
+      self.assertFalse(os.path.isfile('Example_1.0.0_arm64.apk'))
+      Clean('Example', '1.0.0')
+
+      cmd = ['python', 'make_apk.py', '--name=Example', '--app-version=1.0.0',
+             '--package=org.xwalk.example', '--app-url=http://www.intel.com',
```

Comment:
how about `http://www.crosswalk-project.org` instead, Crosswalk is open source project